git-ftp

```diff
@@ -996,12 +997,24 @@ download_remote_updates () {
 		mirror_options="$mirror_options -v"
 	fi
 
+	include=""
+	ignoreall=""
+	# Mirror only the files from the diff between the FTP commit and the current branch/commit
+	if [ $PULL_CHANGED_ONLY -eq 1 ] && [ ! -z $CURRENT_BRANCH ]; then
+		ignoreall=" --exclude '.*' --exclude '.*/'"
+		include=`git diff $CURRENT_BRANCH --name-only | awk 'NF' | sed 's/$/"/' | sed 's/^/ --include "/' | tr -d '\r\n'`
```

Comment:
I think it's better to quote it like this:
```bash
include="$(git diff "$CURRENT_BRANCH" --name-only | awk 'NF' | sed 's/$/"/' | sed 's/^/ --include "/' | tr -d '\r\n')"
```

It preserves newlines and doesn't do any shell expansions.